### DISCUSSION TITLE
Update python dependencies (requirements.txt) in one Github repository based on another Github repository

### DISCUSSION THREAD

**Discussion by avik-taranis** [renovatebot/renovate#23576] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I have 2 Github repositories - one is a template repository and another is a repository created from that template.
I want to create pull requests for Python packages updates in the Github repository which was created from the template repository and I want the versions in the template repository to be the latest ones, meaning that Renovate will treat them as the latest versions available and will create PRs with their versions.

Is that possible?

### What you tried so far.

-

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment]
no
